draft-ietf-anima-brski-prm.md

```diff
@@ -2174,122 +2410,141 @@ When using a JWS signature, the Status Query artifact looks as shown in {{stat_r
-* "connect-error": Pledge connection establishment terminated with error.
-  Additional information may be provided in the reason or reason-context.
-  The pledge signs the response message using its LDevID(Pledge).
+As the pledge is assumed to utilize its bootstrapped EE credentials in communication with other peers, additional status information is provided for the connectivity to other peers, which may be helpful in analyzing potential error cases.
 
-The pledge-status responses are cumulative in the sense that connect-success implies enroll-success, which in turn implies voucher-success.
-The reason-context is an arbitrary JSON object that provides additional information specific to a failure. 
-The content of this field is not subject to standardization, but examples are provided in {{stat_res}}. 
+* `connect-success`: Pledge could successfully establish a connection to another peer.
+  The pledge signs the response message using its domain-owner signed EE certificate/credentials.
+* `connect-error`: Pledge connection establishment terminated with error.
+  The pledge signs the response message using its domain-owner signed EE certificate/credentials.
 
-{{stat_res}} provides an example for the bootstrapping-status information.
+The pledge-status responses are cumulative in the sense that connect-success implies enroll-success, which in turn implies voucher-success.
 
+{{stat_example}} below provides an example for bootstrapping status information in the JSON Pledge Status Data:
 
 ~~~~
-# The pledge "status-response" in General JWS Serialization syntax
-{
-  "payload": BASE64URL(UTF8(status-response)),
-  "signatures": [
-    {
-      "protected": BASE64URL(UTF8(JWS Protected Header)),
-      "signature": BASE64URL(JWS Signature)
-    }
-  ]
-}
-
-# Example: Decoded payload "status-response" representation
-  in JSON syntax
 {
   "version": 1,
   "status": "enroll-success",
   "reason-context": {
     "additional" : "JSON"
   }
 }
+~~~~
+{: #stat_example title='JSON Pledge Status Data Example' artwork-align="left"}
 
-# Example: Decoded "JWS Protected Header" representation
-  in JSON syntax
+#### JWS Protected Header
+
+The JWS Protected Header of the pStatus artifact MUST contain the following standard Header Parameters as defined in {{RFC7515}}:
+
+* `alg`: SHALL contain the algorithm type used to create the signature, e.g., `ES256`, as defined in {{Section 4.1.1 of RFC7515}}
+* `x5c`: SHALL contain the base64-encoded pledge EE certificate used to sign the pStatus artifact;
+  it SHOULD also contain the certificate chain for this certificate;
+  if the certificate chain is not included in the x5c Header Parameter, it MUST be available at the Registrar-Agent for verification
+
+{{pstatus_header}} below shows an example for this JWS Protected Header:
+
+~~~~
 {
   "alg": "ES256",
   "x5c": [
     "base64encodedvalue==",
     "base64encodedvalue=="
-  ],
-  "typ": "jose+json
```

Comment:
I found it weird that the `typ` was only included in this example and not defined for the other jose+json artifacts.
I am not sure if there is even a benefit when the default media type is used.

Reply:
I think we had flexibility in mind, if other media types are used. But as BRSKI-PRM uses only the already stated ones, I'm fine with deleting it. It may be enhanced by other drafts if necessary.

Reply:
Done